src/styles/common/_buttons.scss

```diff
@@ -0,0 +1,43 @@
+@import "../settings";
+@import "../colours";
+
+.button {
+  svg {
+    width: 0.9rem;
+    height: 0.9rem;
+    margin-top: -0.2rem;
+    margin-right: 0.5rem;
+    vertical-align: middle;
+    fill: $colour-white;
+  }
+}
+
+.button.secondary {
```

Comment:
Would `.button.secondary` and `.button.link-button` be more "BEM"-like if made as children of `.button`?